keyboards/latinpad/info.json

```diff
@@ -0,0 +1,56 @@
+{
+    "keyboard_name": "latinpad",
+    "url": "https://github.com/latincompass/qmk_firmware/tree/master/keyboards/hasupad",
```

Comment:
This is not the URL to your Open Source repo for the keyboard design. Please change it to your board design.